weave/wandb_interface/wandb_artifact_pusher.py

```diff
@@ -28,6 +28,7 @@ def write_artifact_to_wandb(
     additional_aliases: list = [],
     *,
     _lite_run: typing.Optional[InMemoryLazyLiteRun] = None,
+    merge: typing.Optional[bool] = False,
```

Comment:
2 comments: 
1. `merge` is a confusing param name because it implies that `write_artifact_to_wandb` knows what it means to "merge" something. I think it would be cleaner to be explicit: `assume_artifact_type_exists`.
2. stylistically, it would be better to put non-underscore params before underscore ones.

Reply:
done